### PR TITLE
Hindi corrections

### PR DESCRIPTION
**Pull request by onisereru**

### CODE REVIEW

File: src/hindi.ts [hunk 1]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
   accountAlreadyExists: ` खाता पहले से ही मौजूद है`,
   accountDoesNotExist: `खाता मौजूद नहीं है`,
   add: `जोड़ना`,
-  addArt: `कला जोड़ें`,
+  addArt: `चित्र जोड़ें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:12]: कला & चित्र are synonyms

File: src/hindi.ts [hunk 2]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
-  animeClubViewPostAndEarnFromAnimeContent: `Anime.club   - एनीमे कंटेंट से देखें, पोस्ट   करें और कमाएं`,
-  animeCommunity: `एनीमे   समुदाय`,
-  animeOnly: `केवल   एनीमे`,
-  appsVersionIs: `एप्लिकेशन   का संस्करण है`,
-  areYouSureYouWantTo: `क्या   आप यकीन है कि आप चाहते हैं`,
-  art: `कला`,
-  artAssociatedWithAnime: `एनीमे के साथ जुड़ी कला`,
-  artBy: `          द्वारा   कला        `,
-  artCannotBeChanged: `कला को बदला नहीं जा सकता`,
+  addNewArt: `नई चित्र जोड़ें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:15]: कला & चित्र are synonyms

File: src/hindi.ts [hunk 3]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  addPreview: `पूर्व संवीक्षा जोड़ें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:16]: Correction Accepted

File: src/hindi.ts [hunk 4]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  animeArtByArtistsFromAllOverTheWorld: `दुनिया भर के कलाकारों के द्वारा आनिमे चित्रकला। चित्रकला आनिमे से संबंधित होना चाहिए। कला चयन सर्वश्रेष्ठ आनिमे कला की एक अनंत सूची है`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:17]: एनीमे & आनिमे are synonyms

File: src/hindi.ts [hunk 5]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  animeArtSubscribeToArtistsLikeAndCreate: `आनिमे कला - कलाकारों की सदस्यता लें, पसंद करें, और संग्रह बनाएं`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:18]: एनीमे & आनिमे are synonyms

File: src/hindi.ts [hunk 6]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  animeClubTheFirstPlatformToOfferArtistsPaymentsFromViews: `Anime.club - गूगल ऐडसेंस के माध्यम से कलाकारों को भुगतान की पेशकश करने वाला पहला मंच। कलाकारों को उनके कंटेंट को देखकर समर्थन दें या व्यूज़ से कमाई शुरू करें	`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:19]: Correction Accepted

File: src/hindi.ts [hunk 7]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  animeClubViewPostAndEarnFromAnimeContent: `Anime.club   - आनिमे कंटेंट से देखें, पोस्ट करें और कमाएं`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:20]: एनीमे & आनिमे are synonyms

File: src/hindi.ts [hunk 8]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  animeCommunity: `आनिमे समुदाय`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:21]: एनीमे & आनिमे are synonyms

File: src/hindi.ts [hunk 9]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  animeOnly: `केवल आनिमे`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:22]: एनीमे & आनिमे are synonyms

File: src/hindi.ts [hunk 10]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  appsVersionIs: `एप्लिकेशन का संस्करण है`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:23]: It’s not an error, just multiple spaces

File: src/hindi.ts [hunk 11]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  areYouSureYouWantTo: `क्या आपको यकीन है कि आप चाहते हैं`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:24]: Correction Accepted

File: src/hindi.ts [hunk 12]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  art: `चित्र`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:25]: कला & चित्र are synonyms

File: src/hindi.ts [hunk 13]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  artAssociatedWithAnime: `आनिमे के साथ जुड़ी कला`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:26]: एनीमे & आनिमे are synonyms

File: src/hindi.ts [hunk 14]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  artBy: `चित्रकला द्वारा`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:27]: कला & चित्र are synonyms

File: src/hindi.ts [hunk 15]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  artCannotBeChanged: `चित्रकला को बदला नहीं जा सकता`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:28]: कला & चित्र are synonyms

File: src/hindi.ts [hunk 16]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
   authenticationFailed: `प्रमाणीकरण विफल हो गया`,
-  back: `वापस        `,
+  back: `वापस`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:30]: It’s not an error, just multiple spaces

File: src/hindi.ts [hunk 17]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
   blog: `ब्लॉग`,
-  browseArt: `कला ब्राउज़ करें`,
+  browseArt: `चित्रकला ब्राउज़ करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:32]: कला & चित्र are synonyms

File: src/hindi.ts [hunk 18]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
   cancel: `रद्द करें`,
-  cannotStartWithChars: `एस के   साथ शुरू नहीं कर सकते{:"^$*[]{}()?-"!@#%&/,><':;|~\`"        `,
-  changeArt: `कला को बदलो`,
-  changeAvatar: `अवतार बदलो`,
+  cannotStartWithChars: `इस के साथ शुरू नहीं कर सकते{:"^$*[]{}()?-"!@#%&/,><':;|~\`"        `,
```
Comment (helena-shoukhova) [inline src/hindi.ts:34]: Correction Accepted

File: src/hindi.ts [hunk 19]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  changeArt: `चित्रकला बदलें `,
```
Comment (helena-shoukhova) [inline src/hindi.ts:35]: कला & चित्र are synonyms

File: src/hindi.ts [hunk 20]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  changeAvatar: `अवतार बदलें `,
```
Comment (helena-shoukhova) [inline src/hindi.ts:36]: Correction Accepted

File: src/hindi.ts [hunk 21]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
   city: `शहर`,
   clearAll: `सभी को साफ़ करें`,
   close: `बंद करें`,
   comingSoon: `जल्द ही आ रहा…`,
   community: `समुदाय`,
-  confirmAccount: `          खाते   की पुष्टि करें        `,
-  confirmPassword: `          पासवर्ड   की पुष्टि करें        `,
-  copyLink: `(en) Copy link`,
-  couldNotCreateANewRecord: `एक नया रिकॉर्ड नहीं बना सकता`,
+  confirmAccount: `खाते की पुष्टि करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:43]: It’s not an error, just multiple spaces

File: src/hindi.ts [hunk 22]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  confirmPassword: `पासवर्ड की पुष्टि करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:44]: It’s not an error, just multiple spaces

File: src/hindi.ts [hunk 23]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  copyLink: `लिंक कॉपी करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:45]: Correction Accepted

File: src/hindi.ts [hunk 24]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  couldNotCreateANewRecord: `नया रिकॉर्ड नहीं बना सकते`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:46]: Correction Accepted

File: src/hindi.ts [hunk 25]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
   country: `देश`,
-  createANewAccountFailed: `एक नया खाता बनाने में विफल रहा`,
+  createANewAccountFailed: `नया खाता बनाने में विफल रहा`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:48]: Correction Accepted

File: src/hindi.ts [hunk 26]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
   createNewAccount: `नया खाता बनाएं`,
   delete: `हटाएं`,
-  deleteArt: `(en) Delete art`,
-  deletePost: `(en) Delete post`,
+  deleteArt: `चित्रकला हटाएं`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:51]: Correction Accepted

File: src/hindi.ts [hunk 27]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  deletePost: `पोस्ट हटायें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:52]: Correction Accepted

File: src/hindi.ts [hunk 28]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
   description: `वर्णन`,
   details: `विवरण`,
-  didntReceiveEmail: `ईमेल प्राप्त नहीं किया था?`,
-  done: `हो गया`,
-  earningFromArts: `कला से कमाई`,
-  editArt: `कला को संपादित करें`,
+  didntReceiveEmail: `ईमेल प्राप्त नहीं हुआ?`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:55]: Correction Accepted

File: src/hindi.ts [hunk 29]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  done: `समाप्त`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:56]: Correction Accepted

File: src/hindi.ts [hunk 30]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  earningFromArts: `चित्रकला से कमाई`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:57]: Correction Accepted

File: src/hindi.ts [hunk 31]
```diff
@@ -9,53 +9,53 @@ export const text: text = {
...
+  editArt: `चित्रकला को संपादित करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:58]: Correction Accepted

File: src/hindi.ts [hunk 32]
```diff
@@ -64,22 +64,22 @@ export const text: text = {
   errorPleaseContactCustomerSupport: `त्रुटि! कृपया ग्राहक सहायता से संपर्क करें ...`,
   exitEditor: `संपादक से बाहर निकलें`,
   extensions: `एक्सटेंशन`,
-  fileSizeIsTooBig: `फ़ाइल का आकार बहुत बड़ा है`,
+  fileSizeIsTooBig: `फ़ाइल की साइज़ बहुत बड़ी है`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:67]: Correction Accepted

File: src/hindi.ts [hunk 33]
```diff
@@ -64,22 +64,22 @@ export const text: text = {
...
   filter: `फ़िल्टर`,
   filterBy: `द्वारा फ़िल्टर करें`,
-  follow: `          का   पालन करें`,
-  followMeOn: `मुझे का पालन करें पर`,
-  followUsOnTwitter: `ट्विटर पर हमें का पालन करें`,
+  follow: `फॉलो करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:70]: Correction Accepted

File: src/hindi.ts [hunk 34]
```diff
@@ -64,22 +64,22 @@ export const text: text = {
...
+  followMeOn: `मुझे फॉलो करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:71]: Correction Accepted

File: src/hindi.ts [hunk 35]
```diff
@@ -64,22 +64,22 @@ export const text: text = {
...
+  followUsOnTwitter: `ट्विटर पर हमें फॉलो करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:72]: Correction Accepted

File: src/hindi.ts [hunk 36]
```diff
@@ -64,22 +64,22 @@ export const text: text = {
...
   forgotPassword: `पासवर्ड भूल गए`,
   forNewMasterpieces: `नई कृतियों के लिए`,
   four04: `— 404! —`,
   guestView: `अतिथि दृश्य`,
   homepage: `मुखपृष्ठ`,
-  iAgreeWith: `मैं के साथ सहमत हूँ`,
-  id: `(en) ID`,
-  imgHeightNeedsToBeBiggerThanOrEqualTo: `आइएमजी   की ऊंचाई से बड़ा या बराबर होने की जरूरत है`,
-  imgWidthNeedsToBeBiggerThanOrEqualTo: `आईएमजी की चौड़ाई से बड़ा या बराबर होने की जरूरत है`,
+  iAgreeWith: `मैं सहमत हूँ`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:79]: Correction Accepted

File: src/hindi.ts [hunk 37]
```diff
@@ -64,22 +64,22 @@ export const text: text = {
...
+  id: `आईडी`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:80]: Correction Accepted

File: src/hindi.ts [hunk 38]
```diff
@@ -64,22 +64,22 @@ export const text: text = {
...
+  imgHeightNeedsToBeBiggerThanOrEqualTo: `इमेज की ऊंचाई से बड़ा या बराबर होने की जरूरत है`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:81]: Correction Accepted

File: src/hindi.ts [hunk 39]
```diff
@@ -64,22 +64,22 @@ export const text: text = {
...
+  imgWidthNeedsToBeBiggerThanOrEqualTo: `इमेज की चौड़ाई से बड़ा या बराबर होने की जरूरत है`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:82]: Correction Accepted

File: src/hindi.ts [hunk 40]
```diff
@@ -89,7 +89,7 @@ export const text: text = {
   livingIn: `में रहते हैं`,
   logout: `लॉगआउट`,
   maximumNChars: (n) => `अधिकतम ${n} वर्ण`,
-  maxSize: `अधिकतम आकार`,
+  maxSize: `अधिकतम साइज़`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:92]: Correction Accepted

File: src/hindi.ts [hunk 41]
```diff
@@ -98,10 +98,10 @@ export const text: text = {
   need1Lowercase: `1 लोअरकेस की आवश्यकता है`,
   need1Number: `1 नंबर की आवश्यकता है`,
   need1Uppercase: `1 अपरकेस की आवश्यकता है`,
-  needAtLeast1CharacterFromN: (n) => `[${n}] से कम से कम 1 चरित्र की आवश्यकता है`,
-  needNCharacter: (n) => `${n} चरित्र की आवश्यकता है`,
+  needAtLeast1CharacterFromN: (n) => `[${n}] से कम से कम 1 करेक्टर की आवश्यकता है`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:101]: Correction Accepted

File: src/hindi.ts [hunk 42]
```diff
@@ -98,10 +98,10 @@ export const text: text = {
...
+  needNCharacter: (n) => `${n} करेक्टर की आवश्यकता है`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:102]: Correction Accepted

File: src/hindi.ts [hunk 43]
```diff
@@ -98,10 +98,10 @@ export const text: text = {
...
   newAccount: `नया खाता`,
-  newArt: `नई कला`,
+  newArt: `नई चित्रकला`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:104]: Correction Accepted

File: src/hindi.ts [hunk 44]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
   or: `या`,
   password: `पासवर्ड`,
   payments: `भुगतान`,
-  pleaseCheckYour: `कृपया अपनी जांच करें`,
+  pleaseCheckYour: `कृपया जांच करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:120]: Correction Accepted

File: src/hindi.ts [hunk 45]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
-  save: `सहेजें`,
-  saveAvatar: `अवतार सहेजें`,
-  saveChanges: `परिवर्तनों को सहेजें`,
-  saveThumbnail: `थंबनेल सहेजें`,
-  selectAnime: `एनीमे का चयन करें`,
-  selectAvatar: `अवतार का चयन करें`,
-  selectCharacters: `चरित्र का चयन करें`,
-  selectFile: `फ़ाइल का चयन करें`,
-  selectThumbnail: `थंबनेल का चयन करें`,
+  pleaseVerifyYourAccount: `कृपया अपने खाते को परखें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:125]: Correction Accepted

File: src/hindi.ts [hunk 46]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  pressHereTo: `यहाँ प्रेस करने से`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:126]: Correction Accepted

File: src/hindi.ts [hunk 47]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  save: `सेव`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:127]: Correction Accepted

File: src/hindi.ts [hunk 48]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  saveAvatar: `अवतार सेव करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:128]: Correction Accepted

File: src/hindi.ts [hunk 49]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  saveChanges: `परिवर्तनों को सेव करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:129]: Correction Accepted

File: src/hindi.ts [hunk 50]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  saveThumbnail: `थंबनेल को सेव करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:130]: Correction Accepted

File: src/hindi.ts [hunk 51]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  selectAnime: `आनिमे को चुने`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:131]: Correction Accepted

File: src/hindi.ts [hunk 52]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  selectAvatar: `अवतार को चुने`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:132]: Correction Accepted

File: src/hindi.ts [hunk 53]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  selectCharacters: `करेक्टर को चुने`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:133]: Correction Accepted

File: src/hindi.ts [hunk 54]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  selectFile: `फ़ाइल चुने`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:134]: Correction Accepted

File: src/hindi.ts [hunk 55]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  selectThumbnail: `थंबनेल को चुने`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:135]: Correction Accepted

File: src/hindi.ts [hunk 56]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
   sendEmail: `ईमेल भेजें`,
   settings: `सेटिंग्स`,
-  shareYourStory: `अपनी कहानी साझा करें!`,
+  shareYourStory: `अपनी कहानी शेयर करें!`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:138]: Correction Accepted

File: src/hindi.ts [hunk 57]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
   shop: `दुकान`,
   signIn: `साइन इन करें`,
-  skip: `छोड़ें`,
+  skip: `आगे बड़ें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:141]: Correction Accepted

File: src/hindi.ts [hunk 58]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
   socialMedia: `सोशल मीडिया`,
-  spaceCharacterIsNotAllowed: `अंतरिक्ष चरित्र की अनुमति नहीं है`,
-  successChangedPasswordPleaseSignIn: `सफलता! पासवर्ड बदल दिया है. कृपया साइन इन करें`,
-  successCreatedRecord: `सफलता! रिकार्ड बनाया गया!`,
-  successUpdatedRecord: `सफलता! रिकॉर्ड अपडेट किया गया !`,
+  spaceCharacterIsNotAllowed: `स्पेस करेक्टर की अनुमति नहीं है`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:143]: Correction Accepted

File: src/hindi.ts [hunk 59]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  successChangedPasswordPleaseSignIn: `सफलता! पासवर्ड बदल दिया है।  कृपया साइन इन करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:144]: It’s not an error, just punctuation

File: src/hindi.ts [hunk 60]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  successCreatedRecord: `सफलता! रिकार्ड बना दिया गया है। `,
```
Comment (helena-shoukhova) [inline src/hindi.ts:145]: Correction Accepted

File: src/hindi.ts [hunk 61]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  successUpdatedRecord: `सफलता! रिकॉर्ड अपडेट किया गया है । `,
```
Comment (helena-shoukhova) [inline src/hindi.ts:146]: Correction Accepted

File: src/hindi.ts [hunk 62]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
   tellUsAboutYourself: `हमें अपने बारे में बताएं`,
   termsAndConditions: `नियम और शर्तें`,
   termsOfUse: `उपयोग की शर्तें`,
   title: `शीर्षक`,
-  typeYourTextHere: `अपना पाठ यहां टाइप करें...`,
-  uploadAnotherArt: `एक और कला अपलोड करें`,
-  uploadAnotherAvatar: `एक और अवतार अपलोड करें`,
-  uploadArtNow: `अब कला अपलोड करें!`,
-  uploadNew: `नई अपलोड करें`,
+  typeYourTextHere: `अपना अक्षर यहां दर्ज करें...`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:151]: Correction Accepted

File: src/hindi.ts [hunk 63]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  uploadAnotherArt: `एक और चित्रकला अपलोड करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:152]: Correction Accepted

File: src/hindi.ts [hunk 64]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  uploadAnotherAvatar: `कोई और अवतार अपलोड करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:153]: Correction Accepted

File: src/hindi.ts [hunk 65]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  uploadArtNow: `चित्रकला अभी अपलोड करें!`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:154]: Correction Accepted

File: src/hindi.ts [hunk 66]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  uploadNew: `नया अपलोड करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:155]: Correction Accepted

File: src/hindi.ts [hunk 67]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
   uploadNewArt: `नई कला अपलोड करें`,
-  uploadNewPost: `नई पोस्ट अपलोड करें`,
+  uploadNewPost: `नया पोस्ट अपलोड करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:157]: Correction Accepted

File: src/hindi.ts [hunk 68]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
   username: `उपयोगकर्ता नाम`,
   usernameAlreadyExists: ` उपयोगकर्ता नाम पहले से ही मौजूद है`,
-  valueIsNotTheSame: `(en) Value is not the same`,
-  verificationCode: `सत्यापन कोड`,
-  verifyCode: `कोड सत्यापित करें`,
+  valueIsNotTheSame: `दर्ज चीज़े सामान्य नहीं है `,
```
Comment (helena-shoukhova) [inline src/hindi.ts:160]: Correction Accepted

File: src/hindi.ts [hunk 69]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  verificationCode: `वेरीफिकेशन कोड`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:161]: Correction Accepted

File: src/hindi.ts [hunk 70]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  verifyCode: `कोड वेरिफाए करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:162]: Correction Accepted

File: src/hindi.ts [hunk 71]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
   weAreOnInstagram: `हम इंस्टाग्राम पर हैं`,
-  weAreWorkingOnIt: `हम इस पर काम कर रहे हैं`,
-  weDoNotDisplayNSFWContentYet: `हम अभी तक एनएसएफडब्ल्यू सामग्री प्रदर्शित नहीं करते`,
-  weHaveSentTooManyVerificationEmails: `हम भी कई सत्यापन ईमेल भेजा है, कुछ ही मिनटों के बाद पुन: प्रयास करें`,
+  weAreWorkingOnIt: `हम इस पर काम कर रहें हैं`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:164]: Correction Accepted

File: src/hindi.ts [hunk 72]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  weDoNotDisplayNSFWContentYet: `हम अभी तक एनएसएफडब्ल्यू कंटेंट प्रदर्शित नहीं करतें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:165]: Correction Accepted

File: src/hindi.ts [hunk 73]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  weHaveSentTooManyVerificationEmails: `हमने बहुत सारे वेरीफिकेशन मेल भेज दिए है`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:166]: Correction Accepted

File: src/hindi.ts [hunk 74]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
   welcomeToAnimeClub: `Anime.club में आपका स्वागत है!`,
-  weSentAnEmailTo: `हमने एक ईमेल भेजा है:`,
-  weSentYouANewVerificationEmailPleaseEnterTheCode: `हमने आपको एक नया सत्यापन ईमेल भेजा है। कृपया कोड दर्ज करें`,
+  weSentAnEmailTo: `हमने इस पर ईमेल भेजा है:`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:168]: Correction Accepted

File: src/hindi.ts [hunk 75]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  weSentYouANewVerificationEmailPleaseEnterTheCode: `हमने आपको एक नया वेरीफिकेशन ईमेल भेजा है। कृपया कोड दर्ज करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:169]: Correction Accepted

File: src/hindi.ts [hunk 76]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
   weSupportOnlyTheFollowingFileExtensions: `हम केवल निम्नलिखित फ़ाइल एक्सटेंशन का समर्थन करते हैं:`,
   yes: `हाँ`,
   youAreAlreadyAuthenticated: `आप पहले से ही प्रमाणीकृत हैं`,
-  youCanStillNavigateToThePreviouslyOpenedPages: `आप अभी भी पहले खोले गए पृष्ठों पर नेविगेट कर सकते हैं`,
-  youDidNotUpdateAnyValues: `आपने कोई मान अपडेट नहीं किया है`,
-  youHaveNotSavedAnyChanges: `आपने कोई परिवर्तन नहीं सहेजे हैं। छोड़ना?`,
-  yourAccountIsVerifiedPleaseSignIn: `आपका खाता सत्यापित है. कृपया साइन इन करें`,
-  yourArts: `आपकी कलाएँ`,
+  youCanStillNavigateToThePreviouslyOpenedPages: `आप अभी भी पहले खोले गए पेज पर जा सकते हैं`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:173]: Correction Accepted

File: src/hindi.ts [hunk 77]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  youDidNotUpdateAnyValues: `आपने कोई भी वैल्यू अपडेट नहीं करीं है`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:174]: Correction Accepted

File: src/hindi.ts [hunk 78]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  youHaveNotSavedAnyChanges: `आपने कोई परिवर्तन दर्ज नहीं किए हैं`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:175]: Correction Accepted

File: src/hindi.ts [hunk 79]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  yourAccountIsVerifiedPleaseSignIn: `आपका खाता वेरिफाए हो चुका है। कृपया साइन इन करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:176]: Correction Accepted

File: src/hindi.ts [hunk 80]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  yourArts: `आपकी चित्रकलाएँ`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:177]: Correction Accepted

File: src/hindi.ts [hunk 81]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
   yourAvatar: `आपका अवतार`,
   yourBlog: `आपका ब्लॉग`,
-  yourFansAreWaiting: `अपने प्रशंसकों इंतजार कर रहे हैं`,
-  yourSessionHasExpiredPleaseSignInAgain: `आपका सत्र समाप्त हो गया है, कृपया फिर से साइन इन करें`,
-}
\ No newline at end of file
+  yourFansAreWaiting: `अपके प्रशंसक इंतजार कर रहे हैं`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:180]: Correction Accepted

File: src/hindi.ts [hunk 82]
```diff
@@ -117,66 +117,66 @@ export const text: text = {
...
+  yourSessionHasExpiredPleaseSignInAgain: `आपका सेशन समाप्त हो गया है, कृपया फिर से साइन इन करें`,
```
Comment (helena-shoukhova) [inline src/hindi.ts:181]: Correction Accepted

**Review by helena-shoukhova — APPROVED** [review]

**Comment by o-alexandrov** [comment]
Thank you very much @onisereru ❤️ 
And thank you @helena-shoukhova for the confirmation!

File: src/hindi.ts (hunk 1 above)
Comment (onisereru) [reply src/hindi.ts:12]: कला is more like talent. When use the word कला we mean talent and not art. For art its either चित्र or चित्रकला.
Art in English can mean talent at some places

File: src/hindi.ts (hunk 4 above)
Comment (onisereru) [reply src/hindi.ts:17]: एनीमे is a weird pronunciation, its like enime. आनिमे is perfect to the japanese katakana word anime

**Pull request merged**